packages/fxa-auth-server/lib/sentry.js

```diff
@@ -190,6 +192,26 @@ async function configureSentry(server, config, processName = 'key_server') {
   }
 }
 
+/**
+ * Format a Stripe product/plan metadata validation error message for
+ * Sentry to include as much detail as possible about what metadata
+ * failed validation and in what way.
+ *
+ * @param {string} planId
+ * @param {string | import('@hapi/joi').ValidationError} error
+ */
+function formatMetadataValidationErrorMessage(planId, error) {
+  let msg = `${planId} metadata invalid:`;
+  if (typeof error === 'string') {
+    msg = msg + ' ' + error;
+  } else {
+    for (const errorItem of error.details) {
+      msg = msg + ' ' + errorItem.message + ';';
+    }
```

Comment:
code style comment: ``error.details.map(x => ` ${x};`).join('')`` instead of a for loop.

Reply:
I can do this; just need to add the message prefix and destructure the `error.details` object, which is itself an object literal.

Reply:
oh haha, yeah... thanks for catch my bug in my comment 😓